transformers/events/vat_file/ilk/converter_test.go

```diff
@@ -18,50 +18,74 @@ package ilk_test
 
 import (
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/makerdao/vulcanizedb/pkg/core"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/events/vat_file/ilk"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Vat file ilk converter", func() {
-	var converter = ilk.VatFileIlkConverter{}
+	var converter = ilk.Converter{}
+	db := test_config.NewTestDB(test_config.NewTestNode())
```

Comment:
I'd be a proponent of adding a `CleanTestDB` call in a `BeforeEach` in tests where we're using the database